airbyte-integrations/connectors/source-google-pagespeed-insights/acceptance-test-config.yml

```diff
@@ -35,4 +35,8 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
         ignored_fields:
-          pagespeed: ["lighthouseResult", "analysisUTCTimestamp"] # This differs between two runs, as they are newly generated per run
+          pagespeed:
+            - name: lighthouseResult
+              bypass_reason: "This differs between two runs, as they are newly generated per run"
+            - name: analysisUTCTimestamp
+              bypass_reason: "This differs between two runs, as they are newly generated per run"
```

Comment:
This seems legit for `analysisUTCTimestamp`, why is it the case for `lighthouseResult`?